recipes/flint/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "flint" %}
```

Comment:
Is there a different name we could use. There are multiple projects that use this name. We also already have a `libflint` package.

Reply:
I've renamed it to `flint-checker`, which has been used by other package managers to avoid this exact issue with name conflicts.